src/styles/_button.scss

```diff
@@ -0,0 +1,24 @@
+a {
```

Comment:
Use a class mentioned above, you don;t want to just style all of your `a` tags on the whole page this way.